iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_data.inl

```diff
@@ -11,19 +11,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-#include "iceoryx_posh/internal/popo/building_blocks/chunk_receiver_data.hpp"
+#ifndef IOX_POSH_POPO_BUILDING_BLOCKS_CHUNK_QUEUE_DATA_INL
+#define IOX_POSH_POPO_BUILDING_BLOCKS_CHUNK_QUEUE_DATA_INL
 
 namespace iox
 {
 namespace popo
 {
-ChunkReceiverData::ChunkReceiverData(const cxx::VariantQueueTypes queueType,
-                                     const mepoo::MemoryInfo& memoryInfo) noexcept
-    : ChunkQueueData(queueType)
-    , m_memoryInfo(memoryInfo)
+template <typename ChunkQueueProperties>
+inline ChunkQueueData<ChunkQueueProperties>::ChunkQueueData(cxx::VariantQueueTypes queueType) noexcept
+    : m_queue(queueType)
```

Comment:
You removed the memoryInfo. I guess it was not used yet but introduced by @elfenpiff for later usage, I guess

Reply:
This was part of @mossmaurice commit in #202.

Reply:
This was already merged... Please check if you roll this back. Now it is no more consistent to the other building blocks
FYI @mossmaurice @elfenpiff

Reply:
Looks like a git mixup:  ...o/building_blocks/chunk_receiver_data.cpp → ...popo/building_blocks/chunk_queue_data.inl 
Chunk queue data never had `memoryInfo` and chunk receiver data still has.